TeamCode/src/main/java/org/firstinspires/ftc/teamcode/GMR/Robot/SubSystems/RobotLift.java

```diff
@@ -0,0 +1,114 @@
+package org.firstinspires.ftc.teamcode.GMR.Robot.SubSystems;
+
+import com.qualcomm.robotcore.hardware.DcMotor;
+import com.qualcomm.robotcore.util.Range;
+
+import org.firstinspires.ftc.robotcore.external.Telemetry;
+
+public class RobotLift {
+
+    private DcMotor liftMotor;
+
+    private Telemetry telemetry;
+
+    private boolean encodersCanRun;
+
+    private boolean autoLift;
+
+    private final int LIFT_MIN;
+    private final int LIFT_MAX;
+
+    public RobotLift(DcMotor liftMotor, Telemetry telemetry){
+        this.liftMotor = liftMotor;
+        this.telemetry = telemetry;
+
+        liftMotor.setMode(DcMotor.RunMode.RUN_USING_ENCODER);
+
+        liftMotor.setPower(0);
+
+        this.encodersCanRun = true;
+
+        LIFT_MIN = liftMotor.getCurrentPosition();
+        LIFT_MAX = LIFT_MIN - 3380;
+
+        telemetry.addData("LIFT_MIN", LIFT_MIN);
+        telemetry.addData("LIFT_MAX", LIFT_MAX);
+
+        autoLift = false;
+    }
+
+    public void lift (boolean bumper, float trigger, boolean y, boolean a) {
+        //CONTROLS: Bumper extends lift, trigger retracts lift
```

Comment:
Per Alex, autolift is not working currently - I'm OK merging this without it, but let's add a TODO comment to remind us to fix it later.